src/components/Parameters/Parameters.tsx

```diff
@@ -8,6 +8,7 @@ import { FieldModel, RequestBodyModel } from '../../services/models';
 import { MediaTypesSwitch } from '../MediaTypeSwitch/MediaTypesSwitch';
 import { Schema } from '../Schema';
 
+import { Markdown } from '..';
```

Comment:
Better to specify exact import location:

```suggestion
import { Markdown } from '../Markdown/Markdown';
```

Reply:
Fixed - I amended the commit.